IMC/chatbot/Chatbot.md

```diff
@@ -0,0 +1,167 @@
+1. Type your text query input in the [Dialogflow Simulator](https://cloud.google.com/dialogflow/docs/console#simulator).
+
+<img src="images/6.png" alt="img" style="zoom:70%;" />
+
+
+
+
+**Note: The custom payload of responses may not show up on the Dialogflow Console.**
+
+
+
+![img](images/7.png)
+
+
+
+### Using Dialogflow Messenger
+
+1. Go to the [Dialogflow Console](https://dialogflow.cloud.google.com/).
+2. Select your agent.
+3. Click **Integrations** in the left sidebar menu.
+4. Enable the [**Dialogflow Messenger**](https://cloud.google.com/dialogflow/docs/integrations/dialogflow-messenger) integration.
+5. Paste the embed code shown on the dialog in a webpage on your website. To interact with the agent on your site, click the icon at the bottom right, type your input query, and press Enter. The agent responds.
+6. [Customize the agent](https://cloud.google.com/dialogflow/docs/integrations/dialogflow-messenger#html_customizations) to appear the way you want it.
+
+### Using the Dialogflow API
+
+To interact with your agent using the Dialogflow API, ensure that your Google Cloud Platform project is properly [set up](https://cloud.google.com/dialogflow/docs/quick/setup), then submit a natural language query about COVID-19 and detect its intent.
+
+1. Create a request.json file that includes a natural language query about COVID-19. For example:
+
+   ```json
+    {
+      "query_input": {
+         "text": {
+             "text": "how to get tested",
+             "language_code": "en-US"
+           }
+        }
+      }
+   
+   // See the detectIntent page for details about the JSON format.
+   // page: https://cloud.google.com/dialogflow/docs/reference/rest/v2/projects.agent.sessions/detectIntent#request-body
+   ```
+
+2. Run the following command to send a detect intent request to your agent.
+
+```json
+curl -X POST 
+
+        -H "Authorization: Bearer "$(gcloud auth application-default print-access-token) 
+
+        -H "Content-Type: application/json; charset=utf-8" 
+
+        -d @request.json 
+
+        https://dialogflow.googleapis.com/v2/projects/project-id/agent/sessions/session-id:detectIntent
+```
+
+**Note:** There are many [other ways](https://cloud.google.com/dialogflow/docs/quick/api#detect_intent) (C#, Go, Java, Python, Node.js, PHP, Ruby, etc) to interact with the API.
+
+
+## Conclusion
+
+Congratulations! Now you know how to create a smart Chatbot that gives useful information related to COVID-19. In this tutorial, we showed the easy-to-use tools available such as Google Cloud Platform and Google DiaLog Flow to develop our Chatbot. We used a little bit of coding to user inputs with relevant intents. Remember the framework we used is not just limited to this scenario. In fact, Google cloud services along with DialogFlow can be used to make any sort of information or action Chatbot, from news to personal finances. It is also a fun activity to test the limits of this technology and improve them!
```

Comment:
"We used a little bit of coding to user inputs with relevant intents." is confusing. can you reword it?

Reply:
Please let me know if it makes more sense.